src/wp-includes/class-wp-term-query.php

```diff
@@ -748,56 +742,61 @@ public function get_terms() {
 		$cache_key    = "get_terms:$key:$last_changed";
 		$cache        = wp_cache_get( $cache_key, 'terms' );
 		if ( false !== $cache ) {
-			if ( 'all' === $_fields || 'all_with_object_id' === $_fields ) {
-				$cache = $this->populate_terms( $cache );
+			if ( 'ids' === $_fields ) {
+				$term_ids = wp_list_pluck( $cache, 'term_id' );
+				$cache    = array_map( 'intval', $term_ids );
+			} else if ( 'count' !== $_fields ) {
```

Comment:
```suggestion
			} elseif ( 'count' !== $_fields ) {
```

It's preferable to the concatenated version because it behaves consistently. https://www.php.net/manual/en/control-structures.elseif.php

Reply:
I wonder why the coding standards didnt pick this up.

Reply:
Fixed.